README.md

```diff
@@ -55,23 +55,23 @@ python setup.py install
 ```bash
```

Comment:
Just change 0.2 to 0.3. The absolute path is used here because it is easier to express, and the real mount path can be modified by the user.